### PR TITLE
kmmo: Update readme with known issue workaround

### PR DESCRIPTION
**Pull request by hershpa**
Update links to 4.14.
Signed-off-by: Hersh Pathak hersh.pathak@intel.com

### CODE REVIEW

**Review by chaitanya1731 — APPROVED** [review]
lgtm

**Comment by vbedida79** [comment]
LGTM

**Review by vbedida79 — APPROVED** [review]

File: kmmo/README.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ $ oc label node <node_name> intel.feature.node.kubernetes.io/dgpu-canary=true
 ```
 
 3.	Use pre-build mode to deploy the driver container.
+
+Note: Due to a known KMM regression issue, please follow the steps in [Workaround section](https://github.com/intel/intel-technology-enabling-for-openshift/issues/194) before running below command.
```
Comment (uMartinXu) [inline kmmo/README.md:46]: Note: If the KMM Operator version < 2.0.2, please use the steps in the workaround section of https://github.com/intel/intel-technology-enabling-for-openshift/issues/194  to workaround this known issue

Comment (hershpa) [reply kmmo/README.md]: Done

**Review by uMartinXu — APPROVED** [review]

**Pull request merged**